index.d.ts

```diff
@@ -17,6 +17,15 @@ declare namespace dashjs {
         setCalleeNameVisible(flag: boolean): void;
     }
 
+    namespace Debug {
+        const LOG_LEVEL_NONE = 0;
+        const LOG_LEVEL_FATAL = 1;
+        const LOG_LEVEL_ERROR = 2;
+        const LOG_LEVEL_WARNING = 3;
+        const LOG_LEVEL_INFO = 4;
+        const LOG_LEVEL_DEBUG = 5;
+    }
+
```

Comment:
Once I was able to import the package as ESM, I noticed a bunch of type errors due to missing or out-of-sync definitions in `index.d.ts`. I fixed the few that I found, but there may be others.